src/components/Envelope.vue

```diff
@@ -16,7 +16,7 @@
 		@click="onClick">
 		<template #icon>
 			<div
-				v-if="mailbox.isUnified"
```

Comment:
now it will also show in individual mailboxes like *Draft* and *Sent*. This should only be there in any *unified* mailbox like *All inboxes*, *Priority inbox* and *Outbox*.